### PR TITLE
Basic timer tests

### PR DESCRIPTION
**Pull request by henrikrudstrom**
depends #89

### CODE REVIEW

**Comment by Plaristote** [comment]
LGTM

File: tests/QtQuick/qml/TimerSingleshot.qml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.5
+Timer{
```
Comment (ChALkeR) [inline tests/QtQuick/qml/TimerSingleshot.qml]: `Timer {`

**Comment by ChALkeR** [comment]
LGTM

**Comment by ChALkeR** [comment]
Re-opened due to #101.

**Comment by ChALkeR** [comment]
Pushed changes from f44fe52, amended commit message.

**Comment by ChALkeR** [comment]
`this.div` is undefined inside
```js
    expect(function() {
      var div = loader('NamedSignal', this.div).dom;
    }).not.toThrow();
```

**Pull request merged**